Resources/templates/assets/css/modaal.css

```diff
@@ -0,0 +1,661 @@
+/*!
```

Comment:
OK. Grand merci!

Le 03/06/16 15:44, Godfraind a écrit :
>
> In Resources/templates/assets/css/modaal.css 
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65708890>:
>
> > @@ -0,0 +1,661 @@
> > +/*!
>
> Ah mais j'peux pas c'est sur ton dépôt, on le fera ensemble plus tard 
> alors, je merge et fais la maj.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65708890>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ANpV2poNMLSNUl5nH5qgZW2APIMIwxyAks5qIC_UgaJpZM4ItfhJ>.
>